nano/core_test/websocket.cpp

```diff
@@ -383,7 +383,7 @@ TEST (websocket, vote)
 }
 
 /** Tests vote subscription options */
-TEST (websocket, vote_options)
+TEST (websocket, DISABLED_vote_options)
```

Comment:
Why disabling?

Reply:
Ergh good spot, those changes should not have been included, they were remnants from some previous ASAN/TSAN runs I did to get past some errors